benderliz/2021-12-7-Liz'sMeetupWriteup.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
```

Comment:
Liz, could you move this file path into the `_posts` directory? You can edit the file from this PR under the 3-dot ellipsis menu, then click into the title and backspace to get to where you need to edit (before `benderliz/`.

I also recommend removing the apostrophe from the file title.